test/runtest.py

```diff
@@ -2767,3 +2767,14 @@ def f():
         return 1
 
     assert ray.get(f.remote()) == 1
+
+
+def test_non_ascii_comment(shutdown_only):
+    ray.init(num_cpus=1)
```

Comment:
Nit, you can change `shutdown_only` to `ray_start`, then `ray.init(num_cpus=1)` can be removed.